src/routes/chat/ChatSendMessage/ChatSendMessage.jsx

```diff
@@ -1,19 +1,26 @@
 import React, { useCallback } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 
-import { fetchNewMessageAction } from 'reducers/ChatDux';
-import './ChatInput.scss';
+import { reqNewMessage } from 'reducers/ChatDux';
+import './ChatSendMessage.scss';
 
-const ChatInput = () => {
+const ChatSendMessage = () => {
   const dispatch = useDispatch();
   const chatRoomId = useSelector(state => state.chat.chatRoomId);
+  const userType = useSelector(state => state.misc.userType);
   const [value, setMessage] = React.useState('');
 
   const fetchNewMessage = useCallback(
     ({ message }) => {
-      dispatch(fetchNewMessageAction({ chatRoomId, message }));
+      dispatch(
+        reqNewMessage({
```

Comment:
see how easily we can read the previous dispatch and how now the dispatch action does not make much sense anymore?